cmd/minikube/cmd/status.go

```diff
@@ -35,12 +36,21 @@ var statusCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
 		defer api.Close()
-		s, err := cluster.GetHostStatus(api)
+		ms, err := cluster.GetHostStatus(api)
 		if err != nil {
 			glog.Errorln("Error getting machine status:", err)
 			os.Exit(1)
 		}
-		fmt.Fprintln(os.Stdout, s)
+		ls := "N/A"
+		if ms == state.Running.String() {
+			ls, err = cluster.GetLocalkubeStatus(api)
+		}
+		if err != nil {
+			glog.Errorln("Error getting machine status:", err)
+			os.Exit(1)
+		}
+		fmt.Fprintln(os.Stdout, "minikubeVM: "+ms)
```

Comment:
What do you think of building a struct like:

type Status struct {
  MinikubeStatus string
  LocalkubeStatus string
}

Then adding a --format flag which applies this struct to a template string, so users can control the output? We could supply a default template which looks like what you currently have.